src/github.com/matrix-org/dendrite/clientapi/jsonerror/jsonerror.go

```diff
@@ -67,6 +67,18 @@ func NotFound(msg string) *MatrixError {
 	return &MatrixError{"M_NOT_FOUND", msg}
 }
 
+// MissingArgument is an error when the client tries to access a resource
+// without providing an argument that is required.
+func MissingArgument(msg string) *MatrixError {
+	return &MatrixError{"M_MISSING_ARGUMENT", msg}
+}
+
+// InvalidArgumentBody is an error when the client tries to provide an
+// invalid body under a valid argument
```

Comment:
Again, that's more a detail, I'm not sure of the phrasing ("body" makes me think of the whole body of a request). fwiw I'd write it as "to provide an invalid value for a valid argument".